Tasks/SonarQubePostTest/SonarQubePostTest.ps1

```diff
@@ -34,4 +34,23 @@ else
      Write-Warning "Could not find the summary report file $summaryMdPath"
 }
 
+. .\CodeAnalysisFilePathComputation.ps1
+$prcaEnabled = Get-TaskVariable -Context $distributedTaskContext -Name "GitPullRequestCodeAnalysisEnabled"
```

Comment:
Hmm.. I've added a check to make sure sonar-report.json file exists before we do any processing (without sonar-report.json we pretty much can't do anything).. this should be ok?